test/kubetest/client.go

```diff
@@ -16,7 +16,22 @@ limitations under the License.
 
 package kubetest
 
-import "k8s.io/client-go/kubernetes"
+import (
+	"k8s.io/client-go/kubernetes"
+)
+
+func ClientLogsContain(client kubernetes.Interface, command string, logEntries []string, opts *RunOptions) Action {
```

Comment:
what does this do?

Reply:
I assume you want me to add docs above.

Reply:
that, and I actually wanted to know what the function does :laughing:

Reply:
I configured nginx to respond with the `x-remote-user header` received. I then check the client logs, if nginx was reached with the identity header, by reading nginx's response.